CMakeLists.txt

```diff
@@ -620,6 +620,7 @@ endif()
 find_package(ZLIB)
 if(ZLIB_FOUND)
   message(STATUS "Using shared zlib")
+  link_libraries(ZLIB::ZLIB)
```

Comment:
I would have expected that there are more target_link_libraries lines that would need to be changed rather adding this line here.

Reply:
Oops... I'll remove this and test again at some point, but I'm on Linux at the moment.